examples/dwarfdump.rs

```diff
@@ -174,6 +177,7 @@ fn main() {
     opts.optflag("r", "", "print .debug_aranges section");
     opts.optflag("y", "", "print .debug_pubtypes section");
     opts.optflag("", "raw", "print raw data values");
+    opts.optopt("m", "match-units", "print compilation units whose output matches a regex", "REGEX");
```

Comment:
We've been trying to follow libdwarf-dwarfdump's flags, and it already uses `-m` for `.debug_macinfo`.  It has `-u` for matching a CU name, and I think this option is close enough that we could use it here.

Reply:
Done.